install/kubernetes/helm/istio/values-istio-sds-auth.yaml

```diff
@@ -9,7 +9,10 @@ global:
   sds:
     enabled: true
     udsPath: "unix:/var/run/sds/uds_path"
-    useNormalJwt: true
+    # When sds is enabled, use trustworthy JWTs only.
+    # Trustworthy JWTs are available from Kubernetes 1.12 (beta) and later.
+    useNormalJwt: false
```

Comment:
Please use different name. There is nothing 'normal' about those JWT, and the name is inconsistent with what is used in other parts of the code. 

"insecureJWT", "unstrustedJWT", "legacyJWT" - and make sure the comments are very clear that this applies
to JWTs without expiration or audience and should only be used for testing purpose.

Reply:
That applies to the main 'values.yaml' file.

Reply:
Also it doesn't appear that the new JWTs are available by default - Kind with 1.12 didn't appear to support them.
Please check which version of Kind supports it ( since all our tests are moving to kind ), and how are they enabled, and add comments for it ( in main value.yaml )

Reply:
I agree that the name normalJWT is not very descriptive. I'll make the change in a separate PR since it's a different issue. 

As for the KinD support, I've already raised an issue here https://github.com/istio/istio/issues/15607. I'll look into how to support it in KinD.

Reply:
These docs are not exposed to the user in https://istio.io/docs/reference/config/installation-options/. Can you add comments to values.yaml explaining how to enable this and what the requirements are, etc